include/cocaine/framework/message.hpp

```diff
@@ -64,13 +64,11 @@ class decoded_message {
     auto meta() const noexcept -> const std::vector<hpack::header_t>&;
 
     template<class Header>
-    boost::optional<hpack::header_t>
+    boost::optional<const hpack::header_t&>
```

Comment:
Looks like minor API breakage - in most cases only rebuild will be requiered. Besides headers are not widely used now, so I believe it will be ok.